rules/windows/defense_evasion_suspicious_process_creation_calltrace.toml

```
@@ -16,6 +16,43 @@ index = ["winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious Process Creation CallTrace"
+note = """## Triage and analysis
+
+### Investigating Suspicious Process Creation CallTrace
+
+Attackers may inject code into child processes' memory to hide its actual activity, evade detection mechanisms, and
+decrease discoverability during forensics. This rule looks for a spawned process by MS office, scripting, and command
+line applications, followed by a process access event for an unknown memory region by the parent process, which can
+indicate a code injection or hollowing attempt.
```

I think "its" is pointing back to "code," but with the other plural nouns it just sounds awkward to me. It could be "their" as in "the attackers' actual activity..."
```suggestion
Attackers may inject code into child processes' memory to hide their actual activity, evade detection mechanisms, and
decrease discoverability during forensics. This rule looks for a spawned process by Microsoft Office, scripting, and command
line applications, followed by a process access event for an unknown memory region by the parent process, which can
indicate a code injection or hollowing attempt.
```